source/jormungandr/jormungandr/scenarios/helper_classes/pt_journey.py

```diff
@@ -124,8 +126,48 @@ def _do_request(self):
         )
         return resp
 
+    def _do_isochrone_request(self):
+        logger = logging.getLogger(__name__)
+        fallback_durations_pool = (
+            self._orig_fallback_durtaions_pool
+            if self._orig_fallback_durtaions_pool is not None
+            else self._dest_fallback_durations_pool
+        )
+        mode = self._dep_mode if self._orig_fallback_durtaions_pool else self._arr_mode
+
+        logger.debug("waiting for fallback durations with %s", mode)
+        fallback_duration_status = fallback_durations_pool.wait_and_get(mode)
+
+        logger.debug("requesting public transport journey with dep_mode: %s", mode)
+
+        fallback_durations = {k: v.duration for k, v in fallback_duration_status.items()}
```

Comment:
:warning:  fallback_duration_status can be none

Reply:
It is not tested in _do_journeys_request either :cry: